lib/db/file_handler.rb

```diff
@@ -0,0 +1,32 @@
+require 'json'
+
+class FileHandler
+
+  def initialize(file)
+    @file = file
+  end
+
+  def read
+    file.rewind
+    parse(file.read)
+  end
+
+  def write(contacts)
```

Comment:
Is just calling it `write` in this context misleading?
This method does more than just writing.